WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModelTests.swift

```diff
@@ -77,6 +77,215 @@ final class StoreOnboardingViewModelTests: XCTestCase {
         XCTAssertEqual(sut.tasksForDisplay.count, 4)
     }
 
+    func test_tasksForDisplay_returns_first_3_incomplete_tasks_when_view_all_button_is_visible_in_collapsed_mode() async {
```

Comment:
nit: `view_all_button_is_visible` is more like "then" than "when"? something like:

```suggestion
    func test_tasksForDisplay_returns_first_3_incomplete_tasks_and_viewAllButton_is_visible_with_4_incomplete_tasks_in_collapsed_mode() async {
```

or since there are test cases on `shouldShowViewAllButton` later the mention of "view all button visible" is probably not necessary.

Reply:
> `view_all_button_is_visible` is more like "then" than "when"? something like:

I am afraid that `view_all_button_is_visible` is not `then`. Here we are testing `tasksForDisplay` and it's behaviour changes [based on view all button's visibility](https://github.com/woocommerce/woocommerce-ios/pull/9174/files#diff-71c880a9f74f980fa43772be01c43133abf24d84c81a79946b54d53db72209e2R32).